src/bundle/commitments.rs

```diff
@@ -194,4 +190,45 @@ mod tests {
             "43cfaab1ffcd8d4752e5e7479fd619c769e3ab459b6f10bbba80533608f546b0"
         );
     }
+
+    fn generate_auth_bundle<FL: OrchardFlavor>(
+        bundle_type: BundleType,
+    ) -> AuthorizedBundle<i64, FL> {
+        let rng1 = StdRng::seed_from_u64(6);
+        let rng2 = StdRng::seed_from_u64(7);
+        let pk = ProvingKey::build::<FL>();
+        generate_bundle(bundle_type)
+            .create_proof(&pk, rng1)
+            .unwrap()
+            .prepare(rng2, [0u8; 32])
+            .finalize()
+            .unwrap()
+    }
```

Comment:
You could use only one random generator (you will have to update the auth digest in the following tests)
fn generate_auth_bundle<FL: OrchardFlavor>(
        bundle_type: BundleType,
    ) -> AuthorizedBundle<i64, FL> {
        let mut rng = StdRng::seed_from_u64(6);
        let pk = ProvingKey::build::<FL>();
        generate_bundle(bundle_type)
            .create_proof(&pk, &mut rng)
            .unwrap()
            .prepare(rng, [0u8; 32])
            .finalize()
            .unwrap()
    }

Reply:
Done, thanks! Updated with these changes.